src/DiagnosticMessages.ts

```diff
@@ -608,13 +609,13 @@ export let DiagnosticMessages = {
     })
 };
 
-let allCodes = [] as number[];
+export const DiagnosticCodeMap = {} as Record<keyof (typeof DiagnosticMessages), number>;
```

Comment:
An easier way to look up diagnostic codes without having to call the DiagnosticMessages function